docs/semgrep-ci/packages-in-semgrep-docker.md

```diff
@@ -0,0 +1,29 @@
+---
+slug: packages-in-semgrep-docker
+title: Packages in Semgrep docker
+hide_title: true
+description: Packages included in the latest Semgrep docker image.
+tags:
+  - Semgrep in CI
+---
+
+# Packages in the Semgrep docker image 
+
+## Packages
+
+The [<i class="fas fa-external-link fa-xs"></i> `semgrep/semgrep:latest` docker image](https://hub.docker.com/r/semgrep/semgrep/tags) contains the following packages:
+
+- `bash`
+- `jq`
+- `curl`
+- Python 3.11 (`3.11-alpine` base image)
+
+The Python 3.11 docker includes additional packages (for example, `ncurses`) that can change without notice. To review them, see the following links:
```

Comment:
maybe add after "that can change without Notice. IMPORTANT: you should not rely on their presence in your CI workflow. Only the packages listed above are officially supported and guaranteed to be available in further version of the docker images"